Add a missing PTRACING guard.

Operator << against PayloadTypes is missing on non-PTRACING builds
(src/rtp.cxx:561 on openh323-1.18.0)

Should be forwarded upstream.

 -- Faidon Liambotis <paravoid@debian.org>

--- asterisk-1.4.10.1~dfsg.orig/channels/h323/ast_h323.cxx
+++ asterisk-1.4.10.1~dfsg/channels/h323/ast_h323.cxx
@@ -287,7 +287,9 @@ int MyH323EndPoint::MyMakeCall(const PSt
 		cout << "\t-- " << GetLocalUserName() << " is calling host " << fullAddress << endl;
 		cout << "\t-- Call token is " << (const char *)token << endl;
 		cout << "\t-- Call reference is " << *callReference << endl;
+#ifdef PTRACING
 		cout << "\t-- DTMF Payload is " << connection->dtmfCodec << endl;
+#endif
 	}
 	connection->Unlock();
 	return 0;
@@ -1649,10 +1651,12 @@ void MyH323Connection::OnSendCapabilityS
 				H245_AudioTelephonyEventCapability & atec = cap;
 				atec.m_dynamicRTPPayloadType = dtmfCodec;
 //				on_set_rfc2833_payload(GetCallReference(), (const char *)GetCallToken(), (int)dtmfCodec);
+#ifdef PTRACING
 				if (h323debug) {
 					cout << "\t-- Transmitting RFC2833 on payload " <<
 						atec.m_dynamicRTPPayloadType << endl;
 				}
+#endif
 			}
 		}
 	}
@@ -1712,9 +1716,11 @@ BOOL MyH323Connection::OnReceivedCapabil
 		on_set_rfc2833_payload(GetCallReference(), (const char *)GetCallToken(), (int)pt);
 		if ((dtmfMode == H323_DTMF_RFC2833) && (sendUserInputMode == SendUserInputAsTone))
 			sendUserInputMode = SendUserInputAsInlineRFC2833;
+#ifdef PTRACING
 		if (h323debug) {
 			cout << "\t-- Inbound RFC2833 on payload " << pt << endl;
 		}
+#endif
 	}
 	memset(&prefs, 0, sizeof(prefs));
 	int peer_capabilities = 0;
